doc/user/content/ingest-data/striim.md

```diff
@@ -0,0 +1,161 @@
+        SECURITY PROTOCOL = 'SASL_PLAINTEXT',
+        SASL MECHANISMS = 'SCRAM-SHA-256', -- or `PLAIN` or `SCRAM-SHA-512`
+        SASL USERNAME = 'foo',
+        SASL PASSWORD = SECRET kafka_password
+        SSH TUNNEL ssh_connection
+    );
+
+    CREATE CONNECTION csr_basic TO CONFLUENT SCHEMA REGISTRY (
+        URL 'https://rp-f00000bar.data.vectorized.cloud:30993',
+        USERNAME = 'foo',
+        PASSWORD = SECRET csr_password
+    );
+    ```
+
+2. Next, create a [Kafka source](/sql/create-source/kafka/):
+
+    ```sql
+    CREATE SOURCE src
+        FROM KAFKA CONNECTION kafka_conn (TOPIC 'striim_topic')
+        KEY FORMAT TEXT
+        VALUE FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn
+        VALUE STRATEGY ID 1
+        ENVELOPE UPSERT;
+    ```
+
+    The `VALUE STRATEGY ID 1` command manually specifies the schema ID, addressing Striim's nonstandard naming conventions in CSR.
+
+    To get the schema ID, use the following command:
+
+    ```sql
+    curl <schema-registry-url>:8081/subjects/<schema-name>/versions/latest | jq .id
+    ```
+
+
+### Alternative Approaches
+
+#### JSON Formatting
+
+Striim’s JSON formatter offers an alternative to Avro, simplifying the configuration process as it does not require a Schema Registry.
+
+Here's how you can leverage JSON formatting with Materialize:
+
+1. **Striim Configuration:**
+   - Set up Striim to transmit events to Kafka in JSON format instead of Avro. This method simplifies the configuration compared to Avro, especially when dealing with complex schema evolution.
+
+2. **Materialize JSON Source Creation:**
+
+    ```sql
+    CREATE SOURCE users
+        FROM KAFKA CONNECTION k (TOPIC 'users')
+        FORMAT JSON
+        ENVELOPE NONE;
+    ```
+
+    This statement creates a source named `users` that reads from the Kafka topic `users` and expects the data to be in JSON format.
+
+3. **Data Transformation in Materialize:**
+
+    ```sql
+    SELECT
+        (r->'data'->>'id')::int4 AS id,
+        (r->'data'->>'email')::text AS email,
+        (r->'data'->>'is_vip')::boolean AS is_vip,
+        (r->'data'->>'created_at')::timestamp AS created_at,
+        (r->'data'->>'updated_at')::timestamp AS updated_at
+    FROM users, jsonb_array_elements(data) r;
+    ```
+
+    This statement transforms the JSON data into a tabular format, allowing you to query and analyze the data.
+
+    {{< json-parser >}}
+
+## Database Specifics
+
+### Oracle
+
+- Oracle versions >= 19c do not support CDDL. Utilize the [Oracle CDC guide](https://www.striim.com/docs/en/oracle-database-cdc.html) for alternative configuration strategies.
+
+### SQL Server
+
+- Implement the [SQL Server CDC guide](https://www.striim.com/docs/en/sql-server-cdc.html) for integration steps specific to SQL Server environments.
+
+## Known Issues
+
+- `BitInt` columns are not supported in Striim's Avro handling. This issue is being addressed by Striim's development team.
```

Comment:
Do we have a link we can toss in here to a bug report?

Reply:
Just added the link to the forum discussion where I've reported that bug